drivers/video/video_ctrls.h

```diff
@@ -54,7 +56,10 @@ struct video_ctrl {
 		int32_t val;
 		int64_t val64;
 	};
-	const char *const *menu;
+	union {
+		const char *const *menu;
+		const int64_t *int_menu;
```

Comment:
Is it making more sense as `int64_menu`? Maybe not as it's to be used for both `int32_t` and `int64_t`.
If some day there is a kind of long int_menu (i.e. calibration data), then having `int_menu` free for it can help.